src/geode/model/helpers/component_mesh_polyhedra.cpp

```diff
@@ -35,50 +35,100 @@
 
 namespace
 {
-    std::vector< std::pair< geode::uuid, geode::PolyhedronVertices > >
-        common_block_vertices(
+    class PolyhedronVerticesPossibilities
+    {
+    public:
+        PolyhedronVerticesPossibilities(
             absl::Span< const std::vector< geode::ComponentMeshVertex > >
                 unique_vertices_cmvs )
-    {
-        if( unique_vertices_cmvs.empty() || unique_vertices_cmvs[0].empty() )
+            : unique_vertices_cmvs_( unique_vertices_cmvs ),
+              nb_unique_vertices_{ static_cast< geode::local_index_t >(
```

Comment:
Should we use a local_index_t? Nothing in the design ensure this

Reply:
We usually use local_index_t when dealing with the vertices of a polyhedron, but I can change it to index_t